arviz/tests/base_tests/test_data.py

```diff
@@ -1246,6 +1249,86 @@ def test_empty_inference_data_object(self):
         assert not os.path.exists(filepath)
 
 
+class TestDataZarr:
```

Comment:
Not sure what would be the best approach here (pinging @canyon289 and @ahartikainen to see what they think), but we should either create a `test_data_zarr.py` or add a class level skipif. As zarr is an optional dependency, we should make sure its test are skipped if it's not installed instead of failing the test suite. Something similar to what we do with numba: https://github.com/arviz-devs/arviz/blob/master/arviz/tests/base_tests/test_diagnostics_numba.py#L16 and bokeh https://github.com/arviz-devs/arviz/blob/master/arviz/tests/base_tests/test_plots_bokeh.py#L48

Reply:
Good Idea was quite easy to add :+1: Haven't created a new file though.

Reply:
Agree with Oriol that a pytest marker should be used to skip tests if zarr isnt present instead of failing the test suite.

 A separate file would be nice